scripts/create_repository.py

```diff
@@ -104,7 +104,9 @@ def map_to_resolved_artifacts(output) -> List[ResolvedArtifact]:
 
 def resolve_artifacts_with_checksums_and_direct_dependencies(root_artifacts) -> List[ResolvedArtifact]:
   command = f'cs resolve {' '.join(root_artifacts)}'
-  output = subprocess.run(command, capture_output=True, text=True, shell=True).stdout.splitlines()
+  proc = subprocess.run(command, capture_output=True, text=True, shell=True)
+  print(proc.stderr)
```

Comment:
I left it here, becouse if some artifact cannot be find (see kind-projector) we're left without any context